pyproject.toml

```diff
@@ -262,7 +262,8 @@ package = "sklearn"  # name of your package
 
 [tool.towncrier]
     package = "sklearn"
-    filename = "doc/whats_new/notes-towncrier.rst"
+    filename = "doc/whats_new/v1.6.rst"
```

Comment:
This is something that we discussed with @lesteve. Basically, we will need to bump this version at each major release for the next one. We can check later if we can automatized this but for the moment I think this is fine.

Reply:
so we should also make sure this step is included in the "how to release guide

Reply:
Yes. My plan is to make an update during the upcoming release process.